tests/unit_tests/regression/test_rotor.cpp

```diff
@@ -49,7 +48,7 @@ void WriteMatrixToFile(const std::vector<std::vector<T>>& data, const std::strin
 
 TEST(RotorTest, IEA15Rotor) {
     // Flag to write output
-    constexpr bool write_output(false);
+    constexpr bool write_output(true);
```

Comment:
did you mean to push this change to main?

Reply:
You're right, I didn't mean to push this.